### PR TITLE
Grouping of IN_TERM features

### PR DESCRIPTION
**Pull request by mkurc-ant**
Added grouping of IN_TERM features so they can be decoded unambiguously by bit2fasm.

The utility `group.py` is a slightly modified version of `fuzzers/034-cmt-pll-pips/fixup_and_group.py`

### CODE REVIEW

File: fuzzers/030-iob/group.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
```
Comment (litghost) [inline fuzzers/030-iob/group.py]: Does this file belong in `030-iob` versus `<root>/utils`?

File: fuzzers/030-iob/Makefile [hunk 2]
```diff
@@ -13,7 +13,8 @@ build/segbits_xiob33.rdb: $(SPECIMENS_OK)
 
 build/segbits_xiob33.db: build/segbits_xiob33.rdb process_rdb.py bits.dbf
 	python3 process_rdb.py build/segbits_xiob33.rdb > build/segbits_xiob33_processed.rdb
-	${XRAY_DBFIXUP} --db-root build --zero-db bits.dbf --seg-fn-in build/segbits_xiob33_processed.rdb --seg-fn-out $@
+	python3 group.py -g tag_groups.txt -i build/segbits_xiob33_processed.rdb -o build/segbits_xiob33_processed.rdb2
+	${XRAY_DBFIXUP} --db-root build --zero-db bits.dbf --seg-fn-in build/segbits_xiob33_processed.rdb2 --seg-fn-out $@
```
Comment (litghost) [inline fuzzers/030-iob/Makefile]: Should the feature set of `group.py` be part of `XRAY_DBFIXUP`?

**Comment by mithro** [comment]
What is the status of this pull request?

File: fuzzers/030-iob/Makefile (hunk 2 above)
Comment (mkurc-ant) [reply fuzzers/030-iob/Makefile]: I moved the `group.py` to the `utils` folder. Do you want this tool to be a part of the dbfixup script? At first I wanted to be that way but I found it difficult to implement the functionality in the dbfixup script architecture.

Comment (litghost) [reply fuzzers/030-iob/Makefile]: Is there a way to have it be a part of the dbfixup script, but only in the lightest sense?  Basically don't try to integrate it complete, but make it an isolated pass within the script?

The idea is future fuzzers might use either the zero-db or the group style?  Thoughts?

Comment (mkurc-ant) [reply fuzzers/030-iob/Makefile]: Hmmm, Ok. I can make it as an additional pass to the db_fixup. So my idea is to keep `group.py` as a separate script but call it from within the db_fixup.

**Comment by litghost** [comment]
@mkurc-ant What is the status of this PR?  The requested change was fairly small?

**Comment by mkurc-ant** [comment]
@litghost I'm integrating the `group.py` with the `dbfixup.py`. Give me time until tomorrow for that.

**Comment by mkurc-ant** [comment]
@litghost I ported the functionality of the `group.py` to `dbfixup.py`. I left the previously written `group.py` script.

**Pull request merged**